@xen-orchestra/fs/src/abstract.js

```diff
@@ -296,6 +319,9 @@ export default class RemoteHandlerAbstract {
     file: File,
     { checksum = false, ...options }: Object = {}
   ): Promise<LaxWritable> {
+    if (typeof file === 'string') {
+      file = normalizePath(file)
+    }
     const path = typeof file === 'string' ? file : file.path
```

Comment:
```js
const path = typeof file === 'string' ? normalizePath(file) : file.path
```

Reply:
Not necessary, `file` has been normalized just above.

Reply:
Whatever you want, but you could remove the condition.

Reply:
That's not enough because `file` is used in the lines below.